backend/curriculum_tracking/tests/test_find_ids_of_latest_reviewers_for_agile_card_api.py

```diff
@@ -0,0 +1,75 @@
+from django.test import TestCase
+from curriculum_tracking.tests.factories import RecruitProjectFactory
+from curriculum_tracking.models import (
+    AgileCard,
+    ContentItem,
+)
+from . import factories
+from core.tests.factories import UserFactory
+from datetime import timedelta
+
+from curriculum_tracking.constants import (
+    NOT_YET_COMPETENT,
+    COMPETENT,
+    EXCELLENT,
+)
+from django.utils import timezone
+
+
+class OldestOpenPrUpdatedTimeTests(TestCase):
+    #card = factories.AgileCardFactory()
+
+    def setUp(self):
+        self.card = factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(
+                content_type=ContentItem.PROJECT,
+                project_submission_type=ContentItem.LINK,
+            ),
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.project = RecruitProjectFactory(content_item=self.card.content_item)
+        self.user = UserFactory()
+        self.card.assignees.set([self.user])
+        self.assertIsNotNone(self.card.recruit_project)
```

Comment:
The card and the recruit_project are not associated with each other, even though they have the same content item.

Eg if Tshepo and Mpho both have cards for simple calc part 1, they each have a card, each card is associated with a different RecruitProject. Both the projects are associated with the same content item

This would be cleaner:

```
self.card = factories.AgileCardFactory(
            content_item=factories.ContentItemFactory(
                content_type=ContentItem.PROJECT,
                project_submission_type=ContentItem.LINK,
            ),
            status=AgileCard.IN_PROGRESS,
        )

self.project = self.card.recruit_project # this should already exist on the card  
self.user = self.card.assignees.first()    # same
```